README.md

```diff
@@ -95,3 +95,24 @@ List of every HackerOoT contributors, from most recent to oldest contribution:
 - ariahiro64
 - ghost
 - krm01
+- Sauraen
+
+This repository contains sauraen's improved animation morphing code, which can be found [here](https://github.com/sauraen/OoTAnimInterp/blob/master/animinterp.c). The following is the copyright notice pertaining to said code:
+
+```
+Copyright (C) 2021 Sauraen
+You may redistribute, create derivative works of, and otherwise copy this
+software, subject to the following conditions:
+- If the software or modified versions of it are distributed in source form,
+  it must retain this copyright notice and list of conditions
+- Sauraen must be credited (e.g. in the Special Thanks) of any project using
+  this software or modified versions of it
+
+This software does not contain any information derived from the Gigaleak.
```

Comment:
This kind of note need not be reproduced here. It's self-evident by the fact it's even considered for inclusion into this repo

Reply:
This is not a part of the copyright notice